<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>613</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Création de la partie de Splendor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Configurationdelapartie">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Configuration de la partie</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="nbJoueurs">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="NombreDeJoueurs">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Nombre de joueurs : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonLancer">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Lancer la partie </string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ActiverUneExtension">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Activer une extension : </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>99</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Oui</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>99</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Non</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChoirExtension">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Choisir l'extension à activer </string>
    </property>
   </widget>
   <widget class="QLabel" name="Joueur1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Joueur 1 </string>
    </property>
   </widget>
   <widget class="QLabel" name="Joueur2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Joueur 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Joueur3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Joueur 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="Joueur4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Joueur 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="J2bot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Bot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="J3bot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Bot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="J4bot">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Bot</string>
    </property>
   </widget>
   <widget class="QComboBox" name="extension">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Cities</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>The Orient</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Trading Posts</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Strongholds</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="nomJoueur1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomJoueur2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomJoueur3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomJoueur4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="avertissement">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
